pontos/github/actions/core.py

```diff
@@ -17,8 +17,11 @@
 
 import contextlib
 import os
+from pathlib import Path
 from typing import Optional
 
+from .errors import GitHubActionsError
```

Comment:
```suggestion
from pontos.github.actions.errors import GitHubActionsError
```

I am a fan of absolute imports btw. :D